docs/get-started/v2/get-started/100ms-dashboard/export-import-template.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Export/Import Templates
+nav: 7.1
+---
+
+Export templates directly from the dashboard and import them into any workspace or account.
+
+## How-to guide
+
+<StepsContainer id="import-export-template">
+
+### Export Template
+
+![Export Template](/docs/v2/100ms-dashboard/export-template.png)
+
+1. Click on the **Templates** dropdown in the sidebar, and pick a template you want to export 
+1. Once you've selected your template, click on the three dot menu near the top right
+1. From the dropdown, click on **Export Template** and you're done
+
+You should now see exported template.json in your downloads.
```

Comment:
dont think the name of the file is `template.json`

Reply:
```suggestion
You should now see exported <template-name>.json in your downloads.
```